miner/worker.go

```diff
@@ -224,7 +224,12 @@ func (self *worker) wait() {
 				}
 				self.mux.Post(core.NewMinedBlockEvent{block})
 
-				glog.V(logger.Info).Infof("🔨  Mined block #%v", block.Number())
+				var stale string
+				if block.Hash() == self.chain.LastBlockHash() {
```

Comment:
I don't think this will work since the block-hash-to-mine will be different for other miners due to coinbase, timestamp etc. Rather it should check if the block-we-mined's parentHash is equal to self.chain.CurrentBlock().ParentHash